e2e/infrastructure/TransactionHttp.spec.ts

```diff
@@ -255,7 +255,7 @@ describe('TransactionHttp', () => {
                 Deadline.create(),
                 mosaicId,
                 new MosaicId(UInt64.fromUint(0).toDTO()),
-                UInt64.fromUint(9999),
+                UInt64.fromUint(4444),
```

Comment:
better to introduce a restriction key enum or is that not possible ?

Reply:
> better to introduce a restriction key enum or is that not possible ?

I think the restriction key is free entry. It can be any Uint64 value

Reply:
ah ok, `magic numbers` logic should be avoided. Also, Why the change from 9999 to 4444 ? 

Better to have the value randomized within range to identify collateral magic number problems easier.

Reply:
> ah ok, `magic numbers` logic should be avoided. Also, Why the change from 9999 to 4444 ?
> 
> Better to have the value randomized within range to identify collateral magic number problems easier.
> 
> It's in a unit test so probably not too important but better to change for something that is understood

Yeah, agreed, It's in e2e test . It's trying to match the reference key announced in previous standalone transaction, otherwise this one won't pass. Will update the test value. 👍